### ISSUE TITLE
Exports * statements are being retained when treeshake is set to false

### ISSUE THREAD

**Issue by alan-agius4** [rollup/rollup#3196]
- Rollup Version: 1.26.0 
- Operating System (or Browser): linux x64 / win32 x64
- Node Version: v12.9.11 / v12.10.0

### How Do We Reproduce?
https://github.com/alan-agius4/rollup-bundle-treeshake-error-3196

### Expected Behavior
When `treeshake` is set to `false`, `export *` statements are removed.

### Actual Behavior
When `treeshake` is set to false, `export *` statements are not being removed which causes an invalid bundle.

Example output:
```js
class Foo {
    
}

export * from './public-api';

export { Foo };
```

**Comment by shellscape** [comment]
@alan-agius4 issue template fields are not suggestions, they are required. Please update your issue with the fields you chose to remove.

**Comment by alan-agius4** [comment]
Updated.

**Comment by alan-agius4** [comment]
@shellscape updated

**Comment by lukastaegert** [comment]
Fix at #3209
